docs/dcp2_system_design.rst

```diff
@@ -544,7 +544,7 @@ underscore, and so that a lexicographical sorting reflects both the
 hierarchical relationship between deployments, projects, datasets and snapshots
 as well as the time they were created. ::
 
-    dataset_name = "hca_" , deployment , "_" , [project_id] , "__" , creation_date , ["_" , qualifier]
+    dataset_name = [atlas], "_" , deployment , "_" , [project_id] , "__" , creation_date , ["_" , qualifier]
```

Comment:
Is the `atlas` property optional on purpose, or is it a typo? as in, could there be a dataset name that could be:
`_dev___20210901`?

Reply:
No, it should be non-optional, updated.